src/Kdyby/Redis/DI/RedisExtension.php

```diff
@@ -155,8 +154,13 @@ protected function loadJournal(array $config)
 
 		$builder = $this->getContainerBuilder();
 
+		$constructParams = array(
+				$this->prefix('@client'),
+				(!isset($config['keyPrefix']) ? null : $config['keyPrefix']),
```

Comment:
- braces around such simple condition are not neccesary
- it's better to switch the logic, as the exclamation mark can be easily overlooked and therefore it's not writing it is more error prone
- contants with uppercase please

```php
isset($config['keyPrefix']) ? $config['keyPrefix']) : NULL,
```